config/initializers/active_job_logger_patch.rb

```diff
@@ -5,6 +5,13 @@
   module ActiveJob
     module Logging
       class LogSubscriber
+        def enqueue(event)
+          info do
+            job = event.payload[:job]
+            "Enqueued #{job.class.name} (Job ID: #{job.job_id}) to #{queue_name(event)} at: #{Time.zone.now}"
```

Comment:
Yeah that seems fine to me! cc @brodygov who also had opinions on our log formats